blog/2021-04-30-weekly-report.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "2021-17 Weekly Report"
+author: "JinnyYi"
+author_url: "https://github.com/JinnyYi"
+tags: [updates, weekly-report]
+---
+
+Hello everyone, here is the weekly report for [AOS](https://aos.dev), range from 2021-04-25 to 2021-04-30.
+
+## Library
+
+### Summer 2021 of Open Source Promotion Plan
+
+Open Source Promotion Plan, a.k.a., OSPP, encourage everyone to pay close attention to open source software and open source community, aiming to cultivate and explore more outstanding developers.
+
+This week we applied to participate in OSSP - Summer 2021 and posed the subject . [@Xuanwo](https://github.com/Xuanwo), [@Prnyself](https://github.com/Prnyself) and [@xxchan](https://github.com/xxchan) will take part in the subject, and we will recruit contributors from student groups.
+
+For details, please refer to https://hackmd.io/XcdKgrqFRs6b1CazTxYX7g.
```

Comment:
Use our docs link instead: https://aos.dev/community/events/ospp-summer-2021

Reply:
We can use `<https://aos.dev/community/events/ospp-summer-2021>` to insert a link quickly.

Reply:
I think using a link without `<>` is also ok?